azext_hanaonazure/commands.py

```diff
@@ -9,13 +9,16 @@
 from ._client_factory import cf_hanainstance_groups
 from azure.cli.core.commands import CliCommandType
 
+
 def load_command_table(self, _):
     custom_tmpl = 'azext_hanaonazure.custom#{}'
     custom_type = CliCommandType(operations_tmpl=custom_tmpl)
 
     with self.command_group('hanainstance', client_factory=cf_hanainstance_groups) as g:
         g.custom_command('list', 'list_hanainstance')
-        g.custom_command('show', 'show_hanainstance', exception_handler=empty_on_404)
+        g.custom_command('show', 'show_hanainstance',
+                         exception_handler=empty_on_404)
         g.custom_command('restart', 'restart_hanainstance')
+        g.custom_command('enable-monitoring', 'enable_monitoring_hanainstance')
```

Comment:
Is there going to be a `disable-monitoring`?

Reply:
I believe that that is the eventual goal, but we don't have it now

Reply:
See the comment on the  `monitor` sub-command.